src/app/body/profile/edit-profile/edit-profile.component.html

```diff
@@ -35,7 +35,19 @@ <h2 class="modal-title " id="exampleModalLabel"><strong>Edit Profile</strong> </
                             <div class="col-6 align-middle"><a class="btn btn-primary" id="check_availability" (click)="checkAvailability()" style="background-color: #5c60ec;">check
                                 availability</a></div>
                         </div>
-                        <a *ngIf="userAvailable" class="btn btn-success">Username available</a>
+                        <a *ngIf="userAvailable" >
+                            <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+                                <symbol id="check-circle-fill" fill="currentColor" viewBox="0 0 16 16">
+                                  <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z"/>
+                                </symbol>
+                            </svg>
```

Comment:
What is the use of this svg?

Reply:
with the help of both two svg it shows an success alert with check sign logo

Reply:
I think there are mat icons you can use to do it.

Reply:
Changes done